stress/common/src/main/java/alluxio/stress/client/ClientIOSummary.java

```diff
@@ -0,0 +1,210 @@
+   */
+  public ClientIOSummary() {
+    this(null, null, new HashMap<>(), 0, 0, 0, 0);
+  }
+
+  /**
+   * Creates an instance.
+   *
+   * @param parameters the parameters for the Fuse IO stress bench
+   * @param baseParameters the base parameters for the Fuse IO stress bench
+   * @param nodes the unique ids of all job workers
+   * @param recordStartMs the timestamp starting counting bytes
+   * @param endMs the timestamp that the test ends
+   * @param ioBytes total number of bytes processed by workers
+   * @param ioMBps aggregated throughput data
+   */
+  public ClientIOSummary(ClientIOParameters parameters, BaseParameters baseParameters,
+      Map<String, ClientIOTaskResult> nodes, long recordStartMs, long endMs,
+      long ioBytes, float ioMBps) {
+    mNodeResults = nodes;
+    mParameters = parameters;
+    mBaseParameters = baseParameters;
+    mRecordStartMs = recordStartMs;
+    mEndMs = endMs;
+    mIOBytes = ioBytes;
+    mThroughput = ioMBps;
+    mIOMBps = mThroughput;
+  }
+
+  @Override
+  public GraphGenerator graphGenerator() {
+    return new GraphGenerator();
+  }
+
+  /**
+   * @return Fuse IO stress bench parameters
+   */
+  public ClientIOParameters getParameters() {
+    return mParameters;
+  }
+
+  /**
+   * @param parameters Fuse IO stress bench parameters
+   */
+  public void setParameters(ClientIOParameters parameters) {
+    mParameters = parameters;
+  }
+
+  /**
+   * @return the base parameters
+   */
+  public BaseParameters getBaseParameters() {
+    return mBaseParameters;
+  }
+
+  /**
+   * @param baseParameters the base parameters
+   */
+  public void setBaseParameters(BaseParameters baseParameters) {
+    mBaseParameters = baseParameters;
+  }
+
+  /**
+   * @return the timestamp starting counting bytes (in ms)
+   */
+  public long getRecordStartMs() {
+    return mRecordStartMs;
+  }
+
+  /**
+   * @param recordStartMs the timestamp starting counting bytes (in ms)
+   */
+  public void setRecordStartMs(long recordStartMs) {
+    mRecordStartMs = recordStartMs;
+  }
+
+  /**
+   * @return the timestamp that test ends (in ms)
+   */
+  public long getEndMs() {
+    return mEndMs;
+  }
+
+  /**
+   * @param endMs the timestamp that test ends (in ms)
+   */
+  public void setEndMs(long endMs) {
+    mEndMs = endMs;
+  }
+
+  /**
+   * @return total number of bytes processed during test time
+   */
+  public long getIOBytes() {
+    return mIOBytes;
+  }
+
+  /**
+   * @param ioBytes total number of bytes processed during test time
+   */
+  public void setIOBytes(long ioBytes) {
+    mIOBytes = ioBytes;
+  }
+
+  /**
+   * @return overall throughput (in MB / s)
+   */
+  public float getIOMBps() {
+    return mIOMBps;
+  }
+
+  /**
+   * @param ioMBps overall throughput (in MB / s)
+   */
+  public void setIOMBps(float ioMBps) {
+    mIOMBps = ioMBps;
+  }
+
+  /**
+   * The graph generator for this summary.
+   */
+  public static final class GraphGenerator extends alluxio.stress.GraphGenerator {
```

Comment:
Remove the duplicate implementation in `ClientIOTaskResult`.

Reply:
I moved the "separating different operations and random read" logic into `summary` but keep the actual drawing inside the `TaskResult`, mostly because the graphs are from the `TaskResult`, making more sense to put them together. The `Summary` is simply a combination of a bunch of `TaskResult`s.